analyzers/tests/SonarAnalyzer.Test/TestCases/TooManyLabelsInSwitch.cs

```diff
@@ -15,15 +17,101 @@ public TooManyLabelsInSwitch(int n, MyEnum en)
             switch (n)
             {
                 case 0:
-                    break;
+                {
+                    {
+                        break;
+                    }
+                }
                 default:
                     break;
             }
 
+            switch(n) // Noncompliant
+            {
+                case 0:
+                    {
+                        break;
+                    }
+                case 1:
+                    {
+                        Console.WriteLine("");
+                        break;
```

Comment:
Is this not a one-liner? Are `break` statements considered additional statements?

Reply:
The `break` (as well as `return` and `throw`) statement is ignored when counting lines.

Reply:
Because of the block statement which is more than one line